api/ServiceWorkerGlobalScope.json

```diff
@@ -404,16 +404,21 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/ServiceWorkerGlobalScope/onabortpayment",
           "support": {
-            "chrome": {
-              "version_added": "61",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "#service-worker-payment-apps",
-                  "value_to_set": "Enabled"
-                }
-              ]
-            },
+            "chrome": [
+              {
+                "version_added": "70"
```

Comment:
This matches `AbortPaymentEvent`, good!

Can you update the commit message to reflect this is updating more than the `onmessageerror` entry?